Electrify.SmartMeterUi/Components/Pages/SmartMeterHome.razor.cs

```diff
@@ -91,6 +92,23 @@ private void CheckForNewToastMessage()
                     Type = ToastType.NoConnection
                 });
             }
+
+            if (ConnectionService.InitialConnectionMade)
```

Comment:
If the InitialConnection is made why would you want to attempt a reconnect or am I being stupid?